woocommerce/woocommerce
# [Product Block Editor]: use AdviceCard empty state sections

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR uses the AdviceCard component for the grouped product, replacing the custom implement with it.

Follow-up of #43427

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the Product Editor app
2. Enable the Linked Product flag
  - Enable the WCA plugin
  - Click on `Tools` -> `WCA Test Helper`
  - Click on `Features` tab
  - Toggle on the `product-linked` feature 

<img width="456" alt="Screenshot 2023-12-26 at 11 36 05" src="https://github.com/woocommerce/woocommerce/assets/77539/a065ce77-fdbc-45bd-99dc-0c00e5cad328">

3. Create/edit a product by using the new Product Editor app
4. Ensure to switch the product type to Grouped:

<img width="590" alt="Screenshot 2024-01-09 at 13 59 35" src="https://github.com/woocommerce/woocommerce/assets/77539/f44d851c-981e-42b9-a25b-5e2b73469850">

6. Scroll down to see the **Products in this group** section
7. Confirm you see the empty state implemented via the Advice card:

<img width="696" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/67e3cc9c-227e-4ed1-a704-daa98e41a348">

<img width="655" alt="Screenshot 2024-01-09 at 14 01 42" src="https://github.com/woocommerce/woocommerce/assets/77539/55507982-5d93-4a2c-818a-c9f73ba6bc5e">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
